modules/calib3d/src/calibration.cpp

```diff
@@ -1094,6 +1094,7 @@ CV_IMPL void cvFindExtrinsicCameraParams2( const CvMat* objectPoints,
         else
         {
             // non-planar structure. Use DLT method
+            CV_CheckGE(count, 6, "DLT algorithm needs at least 6 points for pose estimation from 3D-2D point correspondences.");
```

Comment:
Reference for the DLT: [Multiple View Geometry, Richard Hartley and Andrew Zisserman](http://www.robots.ox.ac.uk/~az/tutorials/tutoriala.pdf).

![img1](https://user-images.githubusercontent.com/8229425/58125194-4f1afb00-7c10-11e9-93f1-c932112dc1e2.png)

![img2](https://user-images.githubusercontent.com/8229425/58125203-53dfaf00-7c10-11e9-8a48-91fc1a7d926b.png)